libsolidity/formal/SMTEncoder.cpp

```diff
@@ -744,6 +744,7 @@ void SMTEncoder::initContract(ContractDefinition const& _contract)
 	m_context.pushSolver();
 	createStateVariables(_contract);
 	clearIndices(m_currentContract, nullptr);
+	m_variableUsage.setCurrentContract(_contract);
```

Comment:
We need to pass this information to `VariableUsage` so it knows which function to inline properly